test-utils/runtime/src/system.rs

```diff
@@ -39,19 +38,87 @@ use sp_std::prelude::*;
 const NONCE_OF: &[u8] = b"nonce:";
 const BALANCE_OF: &[u8] = b"balance:";
 
-decl_module! {
-	pub struct Module<T: Config> for enum Call where origin: T::Origin {}
-}
+pub use self::pallet::*;
+
+#[frame_support::pallet]
+mod pallet {
+	use super::*;
+	use frame_support::pallet_prelude::*;
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(PhantomData<T>);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	pub struct ExtrinsicDataPrefix;
+	impl StorageInstance for ExtrinsicDataPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "ExtrinsicData";
+	}
```

Comment:
Again, I don't get why we need this?

Reply:
I'm not sure if anyone else is using this public module directly, so I've kept the changes to a minimum, if you mind I'll use ·pallet::storage· instead and add generic to the storage types.